libs/designsystem/item/src/item.component.spec.ts

```diff
@@ -115,9 +115,10 @@ describe('ItemComponent', () => {
 
       it('should rotate the icon 180deg if the disclosure icon is "arrow-down', async () => {
         spectator.setInput('disclosure', 'arrow-down');
+        spectator.detectChanges();
 
         const icon = spectator.query<HTMLElement>('kirby-icon');
-        await TestHelper.waitForTimeout(); // Wait a tick
+        await TestHelper.waitForTimeout(50); // Wait a tick
```

Comment:
This 50ms value is a bit of a code-smell to me. 

If the issue is that theres no icon yet, the `TestHelper.ionComponentOnReady()` might be useful here instead, to make sure the underlying ion-icon is rendered?